### PR TITLE
chore: replace with MustAccAddressFromBech32

### PR DESCRIPTION
**Pull request by zakir-code**
ref: #105

### CODE REVIEW

File: app/keepers/keepers.go [hunk 1]
```diff
@@ -431,7 +431,7 @@ func NewAppKeeper(
 		AddRoute(bsctypes.ModuleName, crosschainkeeper.NewModuleHandler(appKeepers.BscKeeper)).
 		AddRoute(polygontypes.ModuleName, crosschainkeeper.NewModuleHandler(appKeepers.PolygonKeeper)).
 		AddRoute(avalanchetypes.ModuleName, crosschainkeeper.NewModuleHandler(appKeepers.AvalancheKeeper)).
-		AddRoute(ethtypes.ModuleName, crosschainkeeper.NewModuleHandler(appKeepers.EthKeeper)).
+		//AddRoute(ethtypes.ModuleName, crosschainkeeper.NewModuleHandler(appKeepers.EthKeeper)).
```
Comment (fx0x55) [inline app/keepers/keepers.go:434]: Hi, @zakir-code, why change here?

Comment (zakir-code) [inline app/keepers/keepers.go:434]:
```suggestion
		AddRoute(ethtypes.ModuleName, crosschainkeeper.NewModuleHandler(appKeepers.EthKeeper)).
```

**Comment by fx0x55** [comment]
@zakir-code This err is undefined, please fix it.

https://github.com/FunctionX/fx-core/blob/87071a39783b883eff15d5da49ff65bb19edf988/x/crosschain/keeper/attestation.go#L99C3-L99C6

**Review by fx0x55 — APPROVED** [review]
LGTM, THX

**Pull request merged**